src/test/java/CalculatorTest.java

```diff
@@ -0,0 +1,161 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CalculatorTest {
+
+    @Test
+    public void normalTest() {
+        // Open Chrome browser
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver");
+        WebDriver driver = new ChromeDriver();
+
+        // Open url of calculator page
+        driver.get("https://healthunify.com/bmicalculator/");
+
+        // Input weight
+        driver.findElement(By.name("wg")).sendKeys("50");
+
+        // Input height
+        driver.findElement(By.name("ht")).sendKeys("160");
+
+        // Click on calculate button
+        driver.findElement(By.cssSelector("[value=Calculate]")).click();
+
+
+        // Add check
+        String normalText = "";
+        try {
+            normalText = driver.findElement(By.name("desc")).getAttribute("value");
+        } catch (Exception e) {
+            e.printStackTrace();
+            driver.quit();
+        }
```

Comment:
Please, use this construction only in case when you have exception in code and you asked to handle exception